packages/cfd/src/Containers/cfd-compare-accounts/instruments-icon-with-label.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import TradingInstrumentsIcon from '../../Assets/svgs/trading-instruments';
+
+interface TIconProps {
+    icon:
+        | 'DerivedFX'
+        | 'Synthetics'
+        | 'BasketIndices'
+        | 'Stocks'
+        | 'StockIndices'
+        | 'Commodities'
+        | 'Forex'
+        | 'Cryptocurrencies'
+        | 'ETF';
+    text: string;
+    highlighted: boolean;
+    className?: string;
+}
+
+const InstumentsIconWithLabel: React.FC<TIconProps> = ({ icon, text, highlighted, className }) => {
```

Comment:
At the start of code migration to TS we agreed that we should not use FC. Change it please.